components/Players/PlayerManager.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div>
+  <div ref="playerContainer">
```

Comment:
Why not use the module as a component?: https://www.npmjs.com/package/vue-fullscreen

(Read ``use as component``)

Reply:
You still have to use a ref though? All that would be different is:
```
  <fullscreen ref="playerContainer">
```
vs 
```
  <div ref="playerContainer">
```

Reply:
Maybe using it as a component could save us the use of `@ts-expect-error`?

Reply:
This is the implementation with `<fullscreen>`
````template
<fullscreen ref="playerContainer">
...
</fullscreen
```

```methods
toggleFullScreen(): void {
  // @ts-expect-error - missing types 
  this.$refs.playerContainer?.toggle({
    callback: (fullscreen: boolean) => {
      this.fullScreenVideo = fullscreen;
    }
  });
}
```

I also can't get it to work using <fullscreen>, I when entering fullscreen, I just end up with a grey screen

```

Reply:
Well keep the first version then ;)

Reply:
That it's because, by default, options have wrapping in a div enabled, you must diisable that.

Anyway, yes, let's keep it as is right now..